### PR TITLE
Add watchdog configuration doc

### PR DESCRIPTION
**Pull request by zhifei92**
### Description
Add a `systemd watchdog` user manual for the kubelet, which introduces why to use the `watchdog` and how to use it.
Related issue https://github.com/kubernetes/kubernetes/issues/127460

### Issue

Closes: #

### CODE REVIEW

File: content/en/docs/reference/node/systemd-watchdog.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+content_type: "reference"
+title: Kubelet Systemd Watchdog
+weight: 80
+---
+
+{{< feature-state feature_gate_name="SystemdWatchdog" >}}
```
Comment (dipesh-rawat) [inline content/en/docs/reference/node/systemd-watchdog.md:7]: It seems like there’s an issue with the PR build failing _[(click here)](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/6715156eb41751000823824d)_ due to a missing feature gate description file for 'SystemdWatchdog'.
> 3:37:46 PM: ERROR Invalid feature gate: 'SystemdWatchdog' is not recognized or lacks a matching description file in 'en/docs/reference/command-line-tools-reference/feature-gates'

_Please refer to the documentation on how to add this here: [k8s.io/docs/contribute/new-content/new-features](https://kubernetes.io/docs/contribute/new-content/new-features/#ready-for-review-feature-gates)._

**Review by dipesh-rawat — COMMENTED** [review]
@zhifei92, perhaps we should consider setting the `dev-1.32` branch as the target branch rather than the _main_ branch for this pull request. Since the change is related to the feature targeting beta in v1.32.

File: content/en/docs/reference/node/systemd-watchdog.md (hunk 1 above)
Comment (zhifei92) [reply content/en/docs/reference/node/systemd-watchdog.md:7]: Thank you for your review

**Comment by zhifei92** [comment]
> @zhifei92, perhaps we should consider setting the `dev-1.32` branch as the target branch rather than the _main_ branch for this pull request. Since the change is related to the feature targeting beta in v1.32.

I opened a new PR. https://github.com/kubernetes/website/pull/48465

**Comment by dipesh-rawat** [comment]
> I opened a new PR. https://github.com/kubernetes/website/pull/48465

Based on author's https://github.com/kubernetes/website/pull/48451#issuecomment-2426783976; closing this PR in favor of newly opened PR https://github.com/kubernetes/website/pull/48465.

/close

**Pull request closed without merging**